prow/pubsub/subscriber/subscriber.go

```diff
@@ -192,7 +192,7 @@ func (s *Subscriber) handlePeriodicJob(l *logrus.Entry, msg messageInterface, su
 	}
 
 	// Adds annotations
-	prowJob = pjutil.NewProwJobWithAnnotation(prowJobSpec, periodicJob.Labels, pe.Annotations)
```

Comment:
Does anything use `NewProwJobWithAnnotation` anymore? Can we remove it?

Reply:
Not anymore, in fact it got removed as part of the PR